opm/parser/eclipse/RawDeck/StarToken.cpp

```diff
@@ -24,10 +24,12 @@
 #include <stdexcept>
 #include <cstdlib>
 
+#include <boost/spirit/include/qi.hpp>
+
 #include <opm/parser/eclipse/RawDeck/StarToken.hpp>
 #include <opm/parser/eclipse/Utility/Stringview.hpp>
 
-
+namespace qi = boost::spirit::qi;
```

Comment:
this is unfortunate IMO: I'd prefer if you replaced calls to `qi::` below by the full thing because -- while adding some additional verbosity -- it takes away the mental burden of remembering that `qi::` is actually `boost::spirit::qi`